docs/book/src/faq.md

```diff
@@ -158,36 +158,6 @@ type StructName struct {
 - Users still receive error notifications from the Kubernetes API for invalid `timeField` values.
 - Developers can directly use the parsed TimeField in their code without additional parsing, reducing errors and improving efficiency.
 
-## How to build a bundle with Kubebuilder-based projects to be managed by OLM and/or published in OperatorHub.io?
-
-You’ll need to create an [OLM bundle][olm-bundle-docs] to publish your operator.
-You can use the [Operator SDK][operator-sdk] and the command `operator-sdk generate kustomize manifests` to generate the files.
-After running the above command, you will find the `ClusterServiceVersion (CSV) ` in the path `config/manifests/bases/`. After that, you can run the bundle generation command:
-```
-kustomize build config/manifests | \
-operator-sdk generate bundle \
---version 0.1.0 \
---package my-operator \
---channels alpha \
---default-channel alpha
-
-```
-
-Now you will have the bundle structure such as:
-
-```
-bundle
-├── manifests
-│   ├── app.mydomain.com_myapps.yaml
-│   ├── kubebuilderdemo-controller-manager-metrics-service_v1_service.yaml
-│   ├── kubebuilderdemo-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml
-│   ├── kubebuilderdemo-myapp-admin-role_rbac.authorization.k8s.io_v1_clusterrole.yaml
-│   ├── kubebuilderdemo-myapp-editor-role_rbac.authorization.k8s.io_v1_clusterrole.yaml
-│   ├── kubebuilderdemo-myapp-viewer-role_rbac.authorization.k8s.io_v1_clusterrole.yaml
-│   └── my-example-operator.clusterserviceversion.yaml
-└── metadata
-    └── annotations.yaml
-```
```

Comment:
I am testing it out, and it is not working
1 - In the bundle, I end up with CSV and metrics service only
2 - the bundle should contain either the CRDs for API, rules and etc
Sorry, it is my fault for not trying out during the review.

Reply:
> I am testing it out, and it is not working 1 - In the bundle, I end up with CSV and metrics service only 2 - the bundle should contain either the CRDs for API, rules and etc Sorry, it is my fault for not trying out during the review.

The previous detailed steps included these, so wouldn’t it be better to keep the details as they are?